### ISSUE TITLE
more than 1 answer

### ISSUE THREAD

**Issue by mickyyu96** [HKUST-KnowComp/R-Net#46]
Hello,
How can i make the model to give me top 3 or more answer ?
i want to add MRR metric

**Comment by wzhouad** [comment]
```python
batch, m, n = tf.shape(outer)[0], tf.shape(outer}[1], tf.shape(outer)[2]
outer1 = tf.reshape(outer, [batch, -1])
outer1 = tf.nn.top_k(outer1, 3).indices
self.yp1s = tf.floor(outer1 / n)
self.yp2s = tf.mod(outer1, n)
```

**Comment by mickyyu96** [comment]
thanks
